proxy/src/main/java/com/wavefront/agent/AbstractAgent.java

```diff
@@ -1051,6 +1057,7 @@ private void loadListenerConfigurationFile() throws IOException {
       traceSamplingRate = Double.parseDouble(config.getRawProperty("traceSamplingRate",
           String.valueOf(traceSamplingRate)).trim());
       traceSamplingDuration = config.getNumber("traceSamplingDuration", traceSamplingDuration).intValue();
+      traceDerivedRedMetricsCustomTagKeysProperty = config.getString("traceDerivedRedMetricsCustomTagKeys", traceDerivedRedMetricsCustomTagKeysProperty);
```

Comment:
avoid scrolling for reviews.
make sure all of it is on one line and then use line break and rest of it on 2nd line
better for readability.